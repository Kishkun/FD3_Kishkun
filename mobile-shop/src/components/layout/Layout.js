import React from "react";
import Sidebar from "../sibedar/Sidebar";

const Layout = (props) => {
    let {children} = props;
    return(
        <div className="view-container">
            <div className="container">
                <div className="row">
                    <div className="header"><h1>Mobile Store</h1></div>
                </div>
                <div className="row">
                    <div className="col-md-5 col-lg-3">
                        <Sidebar />
                    </div>
                    <div className="col-md-7 col-lg-9">
                        {children}
                    </div>
                </div>
            </div>
        </div>
    )
};

export default Layout;