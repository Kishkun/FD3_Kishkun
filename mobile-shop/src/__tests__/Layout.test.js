import React from "react";
import {shallow} from 'enzyme';
import Layout from "../components/layout/Layout";

test('Layout test component', () => {

    const wrapper = shallow (
        <Layout />
    );

    expect(wrapper).toMatchSnapshot();
});