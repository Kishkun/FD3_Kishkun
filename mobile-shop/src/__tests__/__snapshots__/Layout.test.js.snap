// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout test component 1`] = `
<div
  className="view-container"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="header"
      >
        <h1>
          Mobile Store
        </h1>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-5 col-lg-3"
      >
        <Sidebar />
      </div>
      <div
        className="col-md-7 col-lg-9"
      />
    </div>
  </div>
</div>
`;
